README.md

```diff
@@ -106,6 +106,9 @@ sudo make install
 - `PREFIX=<PATH>`: Set the prefix of the installation. (Default: `/usr/local`)
 - `BINDIR=<PATH>`: Set the `dunst` executable's path (Default: `${PREFIX}/bin`)
 - `DATADIR=<PATH>`: Set the path for shared files. (Default: `${PREFIX}/share`)
+- `SYSCONFDIR=<PATH>`: Set the base directory for system config files. (Default: `${PREFIX}/etc/xdg`)
```

Comment:
Is `SYSCONFDIR` unused when you set `SYSCONFFILE`?

Reply:
No, `SYSCONFDIR` is being used no matter what. I dropped `SYSCONFFILE` in the code because it did not make much sense, when walking directories and searching for relative paths anyway. I guess, one could say that users who set SYSCONFFILE explicitly might have their reasons, i.e. they do not want it to be installed as a 'live' config. `SYSCONFDIR`, however, dictates the default for `XDG_CONFIG_DIRS`.